action.yaml

```diff
@@ -23,18 +23,22 @@ outputs:
 runs:
   using: "composite"
   steps:
+    - name: Install outdated tool
+      shell: bash
+      run: |
+        yarn global add check-outdated --preferred-cache-folder ${{ inputs.yarn-cache }}
+        echo "$(yarn global bin)" >> $GITHUB_PATH
+
     - name: Install dependencies
       shell: bash
       run: yarn install --no-progress --preferred-cache-folder ${{ inputs.yarn-cache }} --cwd ${{ inputs.working-dir }}
 
     - name: Check pre update outdated
       shell: bash
-      id: get-pre-update-outdated
       run: |
         set +e
-        outdated=$( ! yarn outdated --preferred-cache-folder ${{ inputs.yarn-cache }} --cwd ${{ inputs.working-dir }} | sed '1,6d; $d')
-        outdated="${outdated//$'\n'/'\n'}"
-        echo "::set-output name=outdated::$outdated"
+        cd ${{ inputs.working-dir }}
+        $( ! $(yarn global bin)/check-outdated --depth 1000 --ignore-dev-dependencies --columns name,current,latest,changes > ${{ runner.temp }}/pre-update )
```

Comment:
Putting this in a file, otherwise you'll have loads of outputs which makes the logs more difficult to read.